server/lobbyconnection.py

```diff
@@ -324,7 +326,9 @@ def on_connection_made(self, protocol: QDataStreamProtocol, peername: Address):
                                 raise ClientError('Your ban attempt upset the database: {}'.format(e))
                     else:
                         self._logger.warning('Administrative action: %s closed client for %s', self.player, player)
-                    await player.lobby_connection.kick()
+                    if player.lobby_connection:
```

Comment:
Yea, I guess I should add that explicitly since I'm trying to follow the example in the weakref docs.
https://docs.python.org/3.5/library/weakref.html#weak-reference-objects